src/basics.md

```diff
@@ -1265,6 +1266,32 @@ fn main() {
     println!("Time elapsed in expensive_function() is: {:?}", duration);
 }
 ```
+[ex-convert-datetime-timezone]: #ex-convert-datetime-timezone
+<a name="ex-convert-datetime-timezone"></a>
+## Converting a local time to UTC time and vice versa
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Gets the local time and displays it using [`offset::Local::now`] and then converts it to the UTC format using the [`DateTime::from_utc`] struct method. A time is then converted using the [`offset::FixedOffset`] struct and the UTC time is the converted to UTC+8 and UTC-.
+
+```rust
+extern crate chrono;
+
+use chrono::{DateTime, FixedOffset, Local, Utc};
+
+fn main() {
+    let local_time: DateTime<Local> = Local::now();
+    let utc_time: DateTime<Utc> = DateTime::<Utc>::from_utc(local_time.naive_utc(), Utc);
+    let china_timezone: FixedOffset = FixedOffset::east(28_800);
```

Comment:
To ease the understanding, is it worth to change `28_800` to `8 * 3600`?

Reply:
Looking at the api examples for [`east`](https://docs.rs/chrono/0.4.0/chrono/offset/struct.FixedOffset.html#method.east) I see that the author thinks such information is nontrivial.
So I'm inclined to follow their convention 👍